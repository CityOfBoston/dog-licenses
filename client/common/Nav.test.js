// @flow

// import React from 'react';
// import renderer from 'react-test-renderer';

// // import Cart from '../store/Cart';
// import Nav from './Nav';

// describe('rendering', () => {
//   let cart;

//   beforeEach(() => {
//     cart = new Cart();
//   });

//   it('renders checkout link', () => {
//     expect(
//       renderer.create(<Nav cart={cart} link="checkout" />).toJSON(),
//     ).toMatchSnapshot();
//   });

//   it('renders lookup link', () => {
//     expect(
//       renderer.create(<Nav cart={cart} link="lookup" />).toJSON(),
//     ).toMatchSnapshot();
//   });
// });
