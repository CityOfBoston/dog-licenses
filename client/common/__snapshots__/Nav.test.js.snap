// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders checkout link 1`] = `
<nav
  className="nv-s nv-s--sticky"
  data-jsx={792433154}
>
  <div
    className="nv-s-l bar"
    data-jsx={792433154}
  >
    <a
      className="nv-s-l-b back-link"
      data-jsx={792433154}
      href="/dogs/checkout"
      onClick={[Function]}
    >
      Checkout
    </a>
    <a
      className="cart-link"
      data-jsx={792433154}
      href="/dogs/checkout"
      onClick={[Function]}
    >
      0
    </a>
  </div>
</nav>
`;

exports[`rendering renders lookup link 1`] = `
<nav
  className="nv-s nv-s--sticky"
  data-jsx={792433154}
>
  <div
    className="nv-s-l bar"
    data-jsx={792433154}
  >
    <a
      className="nv-s-l-b back-link"
      data-jsx={792433154}
      href="/dogs"
      onClick={[Function]}
    >
      Back to Lookup
    </a>
    <a
      className="cart-link"
      data-jsx={792433154}
      href="/dogs/checkout"
      onClick={[Function]}
    >
      0
    </a>
  </div>
</nav>
`;
